Formula/youtubeuploader.rb

```diff
@@ -0,0 +1,54 @@
+class Youtubeuploader < Formula
+  desc "Scripted uploads to Youtube"
+  homepage "https://github.com/porjo/youtubeuploader"
+  url "https://github.com/porjo/youtubeuploader/archive/21.05.tar.gz"
+  sha256 "cc087e05e9a31408a6941030cdb933e6e52d619960e765dbb3d91e3661d3dc98"
+  license "Apache-2.0"
+  head "https://github.com/porjo/youtubeuploader.git", branch: "master"
+
+  depends_on "dep" => :build
```

Comment:
Is there a way to build without this? `dep` is deprecated, so we don't want to use that.

Reply:
Yes, the build seems to complete just fine without `dep`.
I'll remove the statement in the next commit.